inspirehep/modules/workflows/tasks/actions.py

```diff
@@ -111,3 +113,10 @@ def is_record_relevant(obj, *args, **kwargs):
                 len(core_keywords) == 0:
             return False
     return True
+
+
+def is_experimental_paper(obj, eng):
+    """Is the record an experimental paper."""
+    categories = get_value(obj.data, "arxiv_eprints.categories", []) + \
+        get_value(obj.data, "subject_terms.term", [])
+    return "hep-ex" in categories or "Experiment-HEP" in categories
```

Comment:
Subject categories where experiment guessing makes sense are:
Experiment-HEP
Experiment-Nucl
Astrophysics
Instrumentation
And probably the primary arXiv category should be one of these.

Collaboration is a good indicator. But since arXiv does not have a collaboration field it is somehow extracted from the author field and doesn't come in standardised form. If the experiment guesser works as good as we believe it's maybe an unnecessary complication to look at the collaboration?
Btw, Kirsten runs a job (nut sure whether it's in bibcheck) to standardise the collaboration field a bit. Maybe that should we integrated into the ingestion workflow?

- Annette


On May 25, 2016, at 2:35 PM, Samuele Kaplun <notifications@github.com<mailto:notifications@github.com>> wrote:


In inspirehep/modules/workflows/tasks/actions.py<https://github.com/inspirehep/inspire-next/pull/1148#discussion_r64564001>:

> @@ -111,3 +113,10 @@ def is_record_relevant(obj, *args, **kwargs):
>                  len(core_keywords) == 0:
>              return False
>      return True
> +
> +
> +def is_experimental_paper(obj, eng):
> +    """Is the record an experimental paper."""
> +    categories = get_value(obj.data, "arxiv_eprints.categories", []) + \
> +        get_value(obj.data, "subject_terms.term", [])
> +    return "hep-ex" in categories or "Experiment-HEP" in categories


Is this the correct criteria?
E.g.
https://inspirehep.net/search?ln=en&ln=en&p=collaboration%3Aatlas+-subject%3Ahep-ex+&of=hb&action_search=Search&sf=earliestdate&so=d&rm=&rg=25&sc=0

shows that there 3K papers from ATLAS that are not hep-ex.

@annetteholtkamp<https://github.com/annetteholtkamp> what do you think? For which INSPIRE/arXiv categories shall we guess the experiment? Shall we be consverative and just opt for the safe bet of hep-ex/Experiment-HEP?

—
You are receiving this because you were mentioned.
Reply to this email directly or view it on GitHub<https://github.com/inspirehep/inspire-next/pull/1148/files/390ae258420bcf8e80c66a85c07f9cc28ce140f4#r64564001>